src/clib/lib/enkf/enkf_main.cpp

```diff
@@ -457,22 +410,10 @@ static void enkf_main_copy_ensemble(const ensemble_config_type *ensemble_config,
     }
 }
 
-void enkf_main_init_current_case_from_existing(enkf_main_type *enkf_main,
-                                               enkf_fs_type *source_case_fs,
-                                               int source_report_step) {
-
-    enkf_fs_type *current_fs = enkf_main_get_fs(enkf_main);
-
-    enkf_main_init_case_from_existing(enkf_main, source_case_fs,
-                                      source_report_step, current_fs);
-}
-
 static void enkf_main_init_current_case_from_existing_custom(
     enkf_main_type *enkf_main, enkf_fs_type *source_case_fs,
-    int source_report_step, std::vector<std::string> &node_list,
-    std::vector<bool> &iactive) {
-
-    enkf_fs_type *current_fs = enkf_main_get_fs(enkf_main);
+    enkf_fs_type *current_fs, int source_report_step,
```

Comment:
Isn't it better to just inline this in the `init_current_case_from_existing_custom` pybind11 def lambda?